src/RapidezStatamicServiceProvider.php

```diff
@@ -213,6 +218,41 @@ public function bootUtilities() : static
         return $this;
     }
 
+    public function bootSitemaps(): static
+    {
+        Schedule::command('rapidez:statamic:generate:sitemap')->twiceDaily(0, 12);
```

Comment:
Not sure if we want to force this from the package, maye put it somewhere in the readme? Or provide a command from `rapidez/sitemap` where this package hooks into. So you'll just get 1 command which you can schedule yourself which generates all sitemaps.

Reply:
Changed this around so it hooks into https://github.com/rapidez/sitemap/pull/2.
See: https://github.com/rapidez/statamic/pull/78#issuecomment-2475852372